src/app/lib/analyticsUtils/staticATITracking/processClientDeviceAndSendStaticBeacon/index.ts

```diff
@@ -67,53 +72,59 @@ export const addProcessClientDeviceAndSendStaticBeaconToWindow = () => {
 
     params.ref = document.referrer || '';
 
-    if (reverbURL) {
-      const processedReverbUrl = reverbURL
+    if (isLiteSite && window.location.search.length) {
```

Comment:
Could we name window.location.search.length, so it's more readable as to why we need to check it exists please?